library/core/src/ffi/c_str.rs

```diff
@@ -875,4 +875,28 @@ mod verify {
             assert!(c_str.is_safe());
         }
     }
+
+    #[kani::proof]
+    #[kani::unwind(32)]
+    fn check_count_bytes() {
+        const MAX_SIZE: usize = 32;
+        let mut string: [u8; MAX_SIZE] = kani::any();
```

Comment:
Can you rename this to `bytes`?

Reply:
Thank you for pointing that out! I've renamed it to `bytes` to better reflect its purpose before it becomes a string.